pkg/rpc/server/server.go

```diff
@@ -2260,7 +2260,10 @@ func (s *Server) packResponse(r *request.In, result interface{}, respErr *respon
 // logRequestError is a request error logger.
 func (s *Server) logRequestError(r *request.Request, jsonErr *response.Error) {
 	logFields := []zap.Field{
-		zap.String("cause", jsonErr.Data),
+		zap.String("cause", jsonErr.Message),
```

Comment:
We usually have an original error in the `Data`, so I'd rather use it for `cause`. `Message` can be omitted if we're to log error code (there is 1:1 mapping there), but it's a trade-off between readability and compactness.

Reply:
Fixed, I think that code is enough. The resulting log looks like:
```
2022-06-24T12:45:29.140+0300	INFO	Error encountered with rpc request	{"code": -32602, "cause": "invalid params", "method": "getnep17transfers", "params": "[123]"}
```